plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/cwc/editor/context/project/ProjectContextProvider.kt

```diff
@@ -113,9 +113,11 @@ class ProjectContextProvider(val project: Project, private val encoderServer: En
                     if (isInitSuccess) {
                         logger.info { "project context index starting" }
                         delay(300)
-                        val isIndexSuccess = index()
-                        if (isIndexSuccess) isIndexComplete.set(true)
-                        return@launch
+                        if (CodeWhispererSettings.getInstance().isProjectContextEnabled()) {
+                            val isIndexSuccess = index()
```

Comment:
Here you need to call index() regardless of whether projectContext is enabled or not. If enabled, call it with config 'all' otherwise config 'default'

Reply:
nah this one doesn't have the latest lsp API

Reply:
it's still the old one which only does chat vector